src/Splat/ServiceLocation/DependencyResolverMixins.cs

```diff
@@ -162,13 +162,38 @@ public static void UnregisterAll<T>(this IMutableDependencyResolver resolver, st
         /// <param name="resolver">The resolver to register the needed service types against.</param>
         public static void InitializeSplat(this IMutableDependencyResolver resolver)
         {
-            resolver.Register(() => new DefaultLogManager(), typeof(ILogManager));
-            resolver.RegisterConstant(new DebugLogger(), typeof(ILogger));
+            RegisterDefaultLogManager(resolver);
+            RegisterLogger(resolver);
+#if !NETSTANDARD
+            RegisterPlatformBitmapLoader(resolver);
+#endif
+        }
+
+        private static void RegisterDefaultLogManager(IMutableDependencyResolver resolver)
+        {
+            if (!resolver.HasRegistration(typeof(ILogManager)))
+            {
+                resolver.Register(() => new DefaultLogManager(), typeof(ILogManager));
+            }
+        }
+
+        private static void RegisterLogger(IMutableDependencyResolver resolver)
```

Comment:
We are eventually planning on changing the init so splitting this up makes that work easier.

eg going for a similar approach as the asp.net host extensions/builder.

Reply:
ok. i'll do some more housekeeping on this